debian/mariadb-server.mariadb.init

```diff
@@ -156,16 +177,22 @@ case "${1:-''}" in
     # Start MariaDB!
     /usr/bin/mariadbd-safe "${@:2}" 2>&1 >/dev/null | $ERR_LOGGER &
 
-    for i in $(seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}"); do
+    for _ in $(seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}")
```

Comment:
is `_` var a bash best practice thing? Just curious.
Also, why not removing `seq` as you did on line 224.

Reply:
For constituency I think as new one also have '_' 
Reason is that new syntax {1..30} does not allow to use variable.. although it could be half minute without variable possibility.

Reply:
> is `_` var a bash best practice thing? Just curious. Also, why not removing `seq` as you did on line 224.

Yes: https://www.shellcheck.net/wiki/SC2034#intentionally-unused-variables

Reply:
Wouldn’t it be more efficient or idiomatic to write
```sh
seq 1 "${MYSQLD_STARTUP_TIMEOUT:-30}"|while read
do
```
This would assign each read line to the implicit value `$REPLY` if we cared about the contents.